tests/containers/nodejs/Dockerfile

```diff
@@ -7,4 +9,6 @@ WORKDIR /tmp
 RUN mkdir /app
 ADD fibonacci.js /app
 
-CMD ["node", "--perf-prof", "--interpreted-frames-native-stack", "/app/fibonacci.js"]
+ENV NODE_RUNTIME_FLAGS ${NODE_RUNTIME_FLAGS}
```

Comment:
Is this needed? the CMD line could just be `node ${NODE_RUNTIME_FLAGS} /app/fibonacci.js`, can't it?

Reply:
build args are available only at build stage, they aren't available when running container, that's why we need to save it to i. e. env

Reply:
Hmm understood